review

```diff
@@ -138,16 +103,15 @@ review() {
     commitMessage=${*:1}
 
     if [ "$lastRevBranch" == "" ] && [ "$commitMessage" == "" ]; then
-        warn "This is your first review to $reviewRemote. You need to provide a commit message as a second argument."
+        warn "This is your first review to $reviewRemote. You need to provide a commit message as an argument."
         exit 1
     fi
 
     compareBranches $reviewBranch $currentBranch
     hr
 
     exe git checkout $reviewBranch
-    # revBranch is assigned
-    createRevBranch $coreBranchName $lastRevBranch
+    revBranch=$(createRevBranch $coreBranchName $lastRevBranch)
```

Comment:
revBranch will be already assigned after createRevBranch

Reply:
It seems that a pure function with return/assignment would be clearer and more readable than using side effects and explaining them to the user with a comment. 🤔

Reply:
Scope local is great. Still not sure about these echoes from functions.

Reply:
Well, that's exactly same as all the git commands we're using – they echo their output and you assign that output to a variable.
Nothing is actually written out to the console, if that's your concern.

Reply:
Well, that's exactly same as all the git commands we're using – they echo their output and you assign that output to a variable.
Nothing is actually written out to the console, if that's your concern.

Reply:
If you evalute with $() it will slurp all echoed messages that were inside, right? So inside createRevBranch there are two puts that are basically colored echoes. Will they be slurped too? User should see an output from these puts.

Reply:
You're right, of course.
Reverted it back to what it was